addons/base_automation/models/base_automation.py

```diff
@@ -291,7 +291,12 @@ def make_onchange(action_rule_id):
             def base_automation_onchange(self):
                 action_rule = self.env['base.automation'].browse(action_rule_id)
                 result = {}
-                server_action = action_rule.action_server_id.with_context(active_model=self._name, onchange_self=self)
+                server_action = action_rule.action_server_id.with_context(
+                    active_model=self._name,
+                    onchange_self=self,
+                    active_id=None,
+                    active_ids=[],
```

Comment:
Just being picky, can you reorder those elements to put all the `active_` stuff close together?
It will help readability.